src/assets/theme/palette.tsx

```diff
@@ -4,6 +4,36 @@ const white = '#FFFFFF';
 const black = '#000000';
 const lightGray = '#f2f2f2';
 
+const chart = {
+  background: white,
+  foreground: black,
+  axis: colors.grey[700],
+  grid: black,
+  area: colors.grey[200],
+  tooltip: {
+    background: colors.grey[900],
+    text: white,
+    shadow: colors.grey[500],
+  },
+  annotation: black,
+
+  // TODO(michael): Not loving this complexity. Perhaps we could just always use regionColor?
+  regionAnnotationStroke: (isActive: boolean, regionColor: string) =>
+    isActive ? regionColor : lightGray,
```

Comment:
The UX mocks for the dark-mode chart always outline (stroke) the region labels with the region color, but the current light-mode charts only use the region color if the region is active, so I had to make this into a function so I could have different logic for the dark charts. 😦